docs/08-module-block-8/02-learning-goals.adoc

```diff
@@ -8,31 +8,29 @@ Die Teilnehmer:innen können die wesentlichen Anwendungsfälle für die Nutzung
 
 - Basis für Data Analytics und Business Intelligence (BI)
 - Test- und Trainingsdaten für das Erstellen und Verbessern von Maschine Learning Modellen
-- Bereitstellen der Ergebnisse analytischer Auswertungen für die Verwendung in operativen Anwendungen (Reverse ETL, Active/"Closed Loop" DWH)
+- Bereitstellen der Ergebnisse analytischer Auswertungen für die Verwendung in operativen Anwendungen (Reverse ETL)
 
 [[LZ-8-2]]
 ==== LZ 8-2 - Repräsentation von Massendaten
-Die Teilnehmer:innen kennen Verfahren für die Darstellung von Massendaten. Sie kennen Verfahren des Query-Processings wie Aggregation, Projektion, Filterung oder Cubes und verstehen, wie diese Verfahren für die Darstellung von Massendaten verwendet werden können. Sie wissen, dass neben den Daten selbst auch Information zu deren Schema, zum Umfang, zur Aktualität, zu den Datenquellen oder zur Qualität angeboten werden können und i.d.R. auch sollten.
+Die Teilnehmer:innen wissen wie die Verfahren des Query-Processings wie Aggregation oder Cubes für die Darstellung von Massendaten verwendet werden können. Ihnen ist bewusst, dass bei analytischen Daten die Vertrauenswürdigkeit eine zentrale Rolle spielt. Sie wissen, dass neben den Daten selbst auch Information zu deren Schema, zum Umfang, zur Aktualität, zu den Datenquellen oder zur Qualität angeboten werden können und wissen, dass neben den Daten selbst auch der weitere Zusammenhang von Bedeutung ist.
 
-Die Teilnehmer:innen kennen Tools zur Visualisierung von Massendaten. Sie kennen verfügbare Arten von Diagrammen (Heatmap, Ranking, Top, ...) und können sie geeignet einsetzen. Sie kennen BI-/Reporting Tools und können diese etwa für Slice & Dice, Drill (Down, Up, Across, Through) oder statistische Analysen einsetzen. Die Teilnehmer:innen wissen, dass auf Basis dieser Tools auch Dashbords oder Notifikationen (Alerts) umgesetzt werden können.
+Die Teilnehmer:innen kennen Tools zur Visualisierung von Massendaten. Sie kennen verfügbare Arten von Diagrammen (Heatmap, Ranking, Top, ...) und können sie geeignet einsetzen. Sie kennen BI-/Reporting Tools und wissen diese etwa für Slice & Dice, Drill (Down, Up, Across, Through) oder statistische Analysen eingesetzt werden können. Die Teilnehmer:innen wissen, dass auf Basis dieser Tools auch Dashbords oder Notifikationen (Alerts) umgesetzt werden können.
```

Comment:
Das "Einsetzen können" in "Sie kennen verfügbare Arten von Diagrammen (Heatmap, Ranking, Top, ...) und können sie geeignet einsetzen." ist nicht erforderlich. Ich würde das jetzt eher so formulieren: "Sie kennen verfügbare Arten von Diagrammen (Heatmap, Ranking, Top, ...) und können diese speziell im Hinblick auf Kosten/Nutzen beurteilen."
Statt "Sie kennen BI-/Reporting Tools und wissen diese etwa für Slice & Dice, Drill (Down, Up, Across, Through) oder statistische Analysen eingesetzt werden können." lieber: "Sie kennen BI-/Reporting Tools und wissen, wie diese für explorative (Slice & Dice, Drill, etc) oder descriptive Analysen eingesetzt werden können."
Statt "Die Teilnehmer:innen wissen, dass auf Basis dieser Tools auch Dashbords oder Notifikationen (Alerts) umgesetzt werden können." lieber: "Die Teilnehmer:innen wissen, dass auf Basis dieser Tools analytische Anwendungen wie Dashbords umgesetzt werden können."